src/workers/hermes_worker.js

```diff
@@ -10,11 +10,18 @@ This Source Code Form is “Incompatible With Secondary Licenses”, as defined
 import PeriodicWorker from './periodic_worker';
 
 export default class HermesWorker extends PeriodicWorker {
-  constructor(logger) {
+  constructor(dataModelEngine, config, logger) {
     super(5000, logger);
+    this.dataModelEngine = dataModelEngine;
+    this.config = config;
+    this.nonce = 0;
```

Comment:
`BundlingIndex` or `BundleSequenceNumber` is IMO a better name, as `nonce` is somewhat reserved for ethereum transactions.